client/src/pages/automation/workflow-executions/components/dialog-content/WorkflowTaskListAccordion.tsx

```diff
@@ -11,113 +10,115 @@ import ReactJson from 'react-json-view';
-                                            endDate?.getTime() -
-                                                startDate.getTime()
-                                        )}
-                                        ms
-                                    </span>
-                                )}
-
-                                <CheckCircledIcon
-                                    className={twMerge(
-                                        'h-5 w-5',
-                                        allTasksCompleted
-                                            ? 'text-green-500'
-                                            : 'text-red-500'
+                return (
+                    workflowTask?.label && (
+                        <AccordionItem key={id} value={id || ''}>
+                            <AccordionTrigger className="flex w-full items-center justify-between border-gray-100 bg-white px-2 py-3 data-[state=closed]:border-b">
+                                <div className="flex items-center text-sm">
+                                    {taskExecution?.componentDefinition
+                                        ?.icon && (
+                                        <InlineSVG
+                                            className="mr-1 h-6 w-6"
+                                            src={
+                                                taskExecution
+                                                    ?.componentDefinition?.icon
+                                            }
+                                        />
                                     )}
-                                />
-                            </div>
-                        </AccordionTrigger>
 
-                        <AccordionContent className="space-y-4 border-b border-gray-100 p-3">
-                            <div className="rounded-lg">
-                                <header className="flex items-center justify-between rounded-md bg-gray-100 px-2 py-1">
-                                    <span className="text-sm font-medium uppercase">
-                                        Input
-                                    </span>
+                                    {workflowTask?.name || workflowTask?.type}
+                                </div>
 
-                                    <span className="text-xs">
-                                        {startDate?.toLocaleString()}
-                                    </span>
-                                </header>
+                                <div className="flex items-center">
+                                    {startDate && endDate && (
+                                        <span className="ml-auto mr-2 text-xs">
+                                            {Math.round(
+                                                endDate?.getTime() -
+                                                    startDate.getTime()
+                                            )}
```

Comment:
Ja bi ovo spremija u varijablu da nemamo 3 reda i da znamo šta točno ovo predstavlja